src/app/shared/services/userMeta/userMeta.service.ts

```diff
@@ -28,6 +28,10 @@ export class UserMetaService {
     return this.userMeta[key];
   }
 
+  getUserMetaUuid() {
```

Comment:
This is needed to avoid a typeError as `getUserMeta()` can now potentially return a boolean, so the result can't be assigned to a `uuid`. There might be a better way around this